stm32-modules/include/vacuum-module/vacuum-module/MPRLL0025PA00001A.hpp

```diff
@@ -4,17 +4,32 @@
 #include <cstdint>
 #include <optional>
 
+#include "firmware/vacuum_pressure_sensor_policy.hpp"
+
 namespace vacuum_pressure_sensor {
 
 template <typename P>
-concept TMF8820Policy = requires(P p, uint16_t dev_addr, uint16_t reg,
-                                 uint16_t size, uint8_t* data) {
-    { p.i2c_read(dev_addr, reg, data, size) } -> std::same_as<RxTxReturn>;
-    { p.i2c_write(dev_addr, reg, data, size) } -> std::same_as<RxTxReturn>;
+concept MPRPolicy = requires(P p, uint16_t dev_addr, uint16_t reg,
+                             uint16_t size, uint8_t* data) {
+    {
+        p.i2c_read(dev_addr, reg, data, size)
+        } -> std::same_as<i2c::hardware::RxTxReturn>;
+    {
+        p.i2c_write(dev_addr, reg, data, size)
+        } -> std::same_as<i2c::hardware::RxTxReturn>;
 };
 
-constexpr uint8_t DEV_ADDRESS = 0x18;
-constexpr uint8_t MEASURE_PRESSURE_BYTE = 0xAA;
+constexpr uint8_t DEV_ADDRESS = 0x18 << 1;
+constexpr uint8_t MEASURE_PRESSURE_COMMAND = 0xAA;
+constexpr uint8_t STATUS_BUSY_FLAG = 0x20;
+constexpr uint32_t OUTPUT_MAX = 15099494;
+constexpr uint32_t OUTPUT_MIN = 1677722;
+constexpr uint32_t OUTPUT_RANGE_COUNTS = OUTPUT_MAX - OUTPUT_MIN;
+// range for this particular model is 0-25 PSI
+constexpr uint16_t PRESSURE_RANGE_PSI = 25;
+
+constexpr uint8_t PRESSURE_FRAME_LEN = 4;
```

Comment:
- Lets make this 10 so we have enough for longer frames if we need to
- Lets create a WRITE_BUFF so i2c_read and i2c_write use seperate buffers